### ISSUE TITLE
Merging inception retrained models

### ISSUE THREAD

**Issue by redserpent7** [tensorflow/tensorflow#6415]
Hi,

I know this might not be the place to ask this but I did not get any replies nor comments on SO so I thought to give it a shot.

I have done my own training of tensorflow inception a while ago, recently I wanted to add new classes to the original model. However, when I retrain tensorflow it generates a new graph and labels files completely separate from the older one.

Is there a way to combine models? Or do I have to retrain again with the old and new classes?

**Comment by aselle** [comment]
@shlens, do you have any suggestions?

**Comment by shlens** [comment]
There is no easy solution to this but it is feasible. There would be several manual steps:
1. Build a new Inception model.
2. Load all of the Tensor's except for the last layer with the `Saver`
3. For the final classification layer, selectively load Tensor's for the classes you wish to preserve or use in your new model making sure to associate each Tensor with the desired class index. For the other classes, allow the model to use randomly initialized weights.
4. Retrain the model probably starting from a lower learning rate.

**Comment by tfboyd** [comment]
Automatically closing due to lack of recent activity. We hope that you were able to resolve it on your own. However, since this is a support issue rather than a bug or feature request, you will probably get more information by posting it on StackOverflow.
